build/docker/zap-full-scan.py

```diff
@@ -420,6 +424,10 @@ def main(argv):
                 # Save the report
                 write_report(base_dir + report_html, zap.core.htmlreport())
 
+            if report_json:
+                # Save the report
+                write_report(base_dir + report_json, zap.core.jsonreport())
```

Comment:
@claudijd are you available to apply this change? (I think it's the only outstanding comment to merge the changes.)